config/phpcs/ZooRoyal/ruleset.xml

```diff
@@ -137,7 +137,10 @@
     <rule ref="SlevomatCodingStandard.ControlStructures.LanguageConstructWithParentheses"/>
     <rule ref="SlevomatCodingStandard.ControlStructures.RequireNullCoalesceEqualOperator"/>
     <rule ref="SlevomatCodingStandard.Exceptions.DeadCatch"/>
+    <rule ref="SlevomatCodingStandard.Exceptions.RequireNonCapturingCatch"/>
     <rule ref="SlevomatCodingStandard.Functions.StrictCall"/>
+    <rule ref="SlevomatCodingStandard.Functions.DisallowNamedArguments"/>
```

Comment:
@rdss-sknott nur fyi zu den named arguments ist mir zumindest 1 Fall aufgefallen, wo sie schon einen kleinen Mehrwert hätten, und zwar in den neuen php8 attributes. Hier mal ein gist
https://gist.github.com/rdss-skirejewski/8367970d63ea77b210e0b6c4eeab84bd
Aber jetzt nicht der ultimative game changer

Reply:
Ja verstehe ich... Allerdings ist jede Funktion/Klasse mit 16 Parametern deutlich überkomplex